src/datalist-ajax.js

```diff
@@ -186,10 +186,22 @@ class AutoComplete extends HTMLElement {
         for (let d = 0; d < data.length && optMax > 0; d++) {
 
           const value = this.resultname ? data[d][this.resultname] : data[d];
+          const res = [];
+          if (this.storekeys){
+              let keys = this.storekeys.split(',');
```

Comment:
You may need to `trim()` the values so `storekeys="id, name, age"` works as expected.